build_tools/scripts/bazel_to_cmake.py

```diff
@@ -167,7 +226,9 @@ def _convert_deps_block(self, **kwargs):
     #    package2::target
     deps = kwargs.get("deps")
     deps_list = [self._convert_target(dep) for dep in deps]
-    deps_list = sorted(list(set(deps_list)))  # Remove duplicates and sort.
+    #deps_list = sorted(list(set(deps_list)))  # Remove duplicates and sort.
```

Comment:
Delete

Reply:
Intentionally left it, since we will change back. Anyway filtering get improved, thus deleted it.